main.go

```diff
@@ -130,7 +154,7 @@ func getTimeout() time.Duration {
 	taskTimeout := os.Getenv("TASK_TIMEOUT")
 	if taskTimeout == "" {
 		log.Println("Default timeout: 30s")
-		timeout, _ := time.ParseDuration("30s")
+		timeout, _ := time.ParseDuration("30m")
```

Comment:
I have a short timeout here (30s) intentionally so that it gets specified explicitly. Each "user" should be configuring it based on their execution expectation.

Alternatively if you thing we should default to _no timeout_ we can consider that.